File: ui/src/services/LevelStats.js
export class LevelStats {
  constructor ({
    _id = '',
    timesPlayed = 0,
    timesPlayedAvg = 0,
    timesPlayedMax = 0,
    wins = 0,
    winsAvg = 0,
    winsMax = 0,
    id = ''
  }) {
    this._id = _id
    this.timesPlayed = timesPlayed
    this.timesPlayedAvg = timesPlayedAvg
    this.timesPlayedMax = timesPlayedMax
    this.wins = wins
    this.winsAvg = winsAvg
    this.winsMax = winsMax
    this.id = id
  }
}

File: app/actors/messages/admin/LevelStats.scala
package actors.messages.admin
import play.api.libs.json._

case class LevelStats(_id: String,
                      timesPlayed: Int,
                      timesPlayedAvg: Double,
                      timesPlayedMax: Int,
                      wins: Int,
                      winsAvg: Double,
                      winsMax: Int,
                      id: String)

object LevelStats {

  implicit val lsL: OFormat[LevelStats] = Json.format[LevelStats]

}
